components/eventing-controller/Makefile

```diff
@@ -92,8 +92,9 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	kustomize build config/default | kubectl delete -f -
 
 CONTROLLER_GEN = $(PROJECT_DIR)/bin/$(shell uname)/controller-gen
+ARCH := $(shell uname)
```

Comment:
i know this was in there before, but can we rename it from ARCH to OS as it is not the architecture, but the os we are interested in

Reply:
done